patches/minecraft/net/minecraft/world/WorldProvider.java.patch

```diff
@@ -65,12 +65,13 @@
      }
  
      @SideOnly(Side.CLIENT)
-@@ -241,6 +216,336 @@
+@@ -241,6 +216,352 @@
          return new WorldBorder();
      }
  
 +    /*======================================= Forge Start =========================================*/
 +    private net.minecraftforge.client.IRenderHandler skyRenderer = null;
++    private net.minecraftforge.client.SkyRenderer theSkyRenderer = new net.minecraftforge.client.SkyRenderer();
```

Comment:
The similarity of the names `skyRenderer` and `theSkyRenderer` could be misleading. Should they not be renamed to make the difference more clear?

Reply:
yeah, also `the` as a prefix is an antiquated MCP-ism, and really shouldn't be there :P

Reply:
Of course.